From patchwork Wed Dec 12 16:36:36 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: kbuild test robot <fengguang.wu@intel.com>
X-Patchwork-Id: 10726695
Return-Path: <kvm-owner@kernel.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 7B2F214BD
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed, 12 Dec 2018 16:37:25 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 68CE9284D1
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed, 12 Dec 2018 16:37:25 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 5B7242B7BC; Wed, 12 Dec 2018 16:37:25 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.9 required=2.0 tests=BAYES_00,MAILING_LIST_MULTI,
	RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 07785284D1
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed, 12 Dec 2018 16:37:25 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1727861AbeLLQhH (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Wed, 12 Dec 2018 11:37:07 -0500
Received: from mga12.intel.com ([192.55.52.136]:45315 "EHLO mga12.intel.com"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S1726358AbeLLQhH (ORCPT <rfc822;kvm@vger.kernel.org>);
        Wed, 12 Dec 2018 11:37:07 -0500
X-Amp-Result: UNKNOWN
X-Amp-Original-Verdict: FILE UNKNOWN
X-Amp-File-Uploaded: False
Received: from fmsmga004.fm.intel.com ([10.253.24.48])
  by fmsmga106.fm.intel.com with ESMTP/TLS/DHE-RSA-AES256-GCM-SHA384;
 12 Dec 2018 08:37:07 -0800
X-ExtLoop1: 1
X-IronPort-AV: E=Sophos;i="5.56,344,1539673200";
   d="scan'208";a="127307977"
Received: from lkp-server01.sh.intel.com (HELO lkp-server01) ([10.239.97.150])
  by fmsmga004.fm.intel.com with ESMTP; 12 Dec 2018 08:37:05 -0800
Received: from kbuild by lkp-server01 with local (Exim 4.89)
        (envelope-from <lkp@intel.com>)
        id 1gX7VR-0000uK-0j; Thu, 13 Dec 2018 00:37:05 +0800
Date: Thu, 13 Dec 2018 00:36:36 +0800
From: kbuild test robot <fengguang.wu@intel.com>
To: Vivek Goyal <vgoyal@redhat.com>
Cc: kbuild-all@01.org, linux-fsdevel@vger.kernel.org,
        linux-kernel@vger.kernel.org, kvm@vger.kernel.org,
        vgoyal@redhat.com, miklos@szeredi.hu, stefanha@redhat.com,
        dgilbert@redhat.com, sweil@redhat.com, swhiteho@redhat.com
Subject: [PATCH] virtio-fs: fix semicolon.cocci warnings
Message-ID: <20181212163636.GA42062@cairo>
References: <20181210171318.16998-17-vgoyal@redhat.com>
MIME-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Disposition: inline
In-Reply-To: <20181210171318.16998-17-vgoyal@redhat.com>
User-Agent: Mutt/1.5.23 (2014-03-12)
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org
X-Virus-Scanned: ClamAV using ClamSMTP

From: kbuild test robot <fengguang.wu@intel.com>

fs/fuse/virtio_fs.c:88:17-18: Unneeded semicolon


 Remove unneeded semicolon.

Generated by: scripts/coccinelle/misc/semicolon.cocci

Fixes: 065b4fe69a2b ("virtio-fs: Add VIRTIO_PCI_CAP_SHARED_MEMORY_CFG and utility to find them")
CC: Dr. David Alan Gilbert <dgilbert@redhat.com>
Signed-off-by: kbuild test robot <fengguang.wu@intel.com>
---

url:    https://github.com/0day-ci/linux/commits/Vivek-Goyal/virtio-fs-shared-file-system-for-virtual-machines/20181211-103034
base:   https://git.kernel.org/pub/scm/linux/kernel/git/mszeredi/fuse.git for-next

 virtio_fs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/fuse/virtio_fs.c
+++ b/fs/fuse/virtio_fs.c
@@ -85,7 +85,7 @@ static int virtio_pci_find_shm_cap(struc
 		        printk(KERN_ERR "%s: shm cap with bad size offset: %d size: %d\n",
                                __func__, pos, cap_len);
                         continue;
-                };
+                }
 
 		pci_read_config_byte(dev, pos + offsetof(struct virtio_pci_shm_cap,
                                                          id),
